fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/util/type/NumberParmBehaviorUtil.java

```diff
@@ -269,14 +284,7 @@ public static int calculateSignificantFigures(BigDecimal original) {
     public static boolean checkIntegerSearchWithSaEb(Prefix prefix, Class<?> resourceType, QueryParameter queryParm,
             BigDecimal originalNumber)
             throws FHIRPersistenceException {
-        boolean isIntegerSearch = false;
-        try {
-            isIntegerSearch = ValueTypesFactory.getValueTypesProcessor().isIntegerSearch(resourceType, queryParm);
-        } catch (FHIRSearchException e) {
-            log.log(Level.INFO, "Caught exception while checking the value types for parameter '"
-                    + queryParm.getCode() + "'; continuing...", e);
-            // do nothing
-        }
+        boolean isIntegerSearch = isIntegerSearch(resourceType, queryParm);
```

Comment:
Are you sure we need to treat integer search as "special"?

It seems like a lot of code just to fail something.  I don't think supporting "eb" and "sa" prefixes for integers would cause anyone any issue.

Reply:
The spec says `sa (starts-after) and eb (ends-before) are not used with integer values but are used for decimals.` which is why we had that check in the first place.  But to me, thats pretty weak.  I recommend removing.

Reply:
FYI - it's a pretty drastic change.  would love to sync back to explain.  I'll save for post 4.0.1

Reply:
removed, and changed per discussion with Lee